cloudbuild-pr.yaml

```diff
@@ -1,41 +1,26 @@
 steps:
-- name: 'gcr.io/cloud-builders/docker'
-  id: pull-app-image-cache
-  args: ['pull', 'gcr.io/$PROJECT_ID/learningequality-studio-app:latest']
-
-- name: 'gcr.io/cloud-builders/docker'
+- name: 'gcr.io/kaniko-project/executor:latest'
   id: build-app-image
-  waitFor: ['pull-app-image-cache']              # don't wait for previous steps
+  waitFor: ['-']              # don't wait for previous steps
   args: [
-    'build',
     '-f', 'docker/Dockerfile.demo',
```

Comment:
I think the valid flag for kaniko is `--dockerfile` instead of `-f`, which is for docker?